arlo_server/routes.py

```diff
@@ -25,12 +25,13 @@
 
 from arlo_server import app
 from arlo_server.auth import (
+    UserType,
     clear_loggedin_user,
     get_loggedin_user,
     get_loggedin_user_record,
     require_audit_admin_for_organization,
     set_loggedin_user,
-    UserType,
+    with_election_access,
```

Comment:
Could we add this to all the routes that have an `election_id` param and call `require_audit_admin_for_organization`?

Reply:
Sure, forgot about other files!